src/rules/nitro/correct-translation-usage.js

```diff
@@ -0,0 +1,301 @@
+  }
+
+  const translateKeyWrapper = node.arguments[0]
+  const objectValuesWrapper = node.arguments[1]
+
+  if (
+    translateKeyWrapper.type !== 'CallExpression' ||
+    translateKeyWrapper.callee.name !== '__'
+  ) {
+    return context.report({ node, messageId: 'tKeyMustBeWrapped' })
+  }
+
+
+  const translateKey = translateKeyWrapper.arguments[0] || null
+
+  if (translateKey === null) {
+    return context.report({ node, messageId: 'tKeyMustBeSpecified' })
+  }
+
+
+  if (!isStringLiteral(translateKey)) {
+    return context.report({ node, messageId: 'tKeyMustBeStringLiteral' })
+  }
+
+  const tKey = translateKey.value
+  const tKeyCopy = getTranslations(context)[tKey]
+
+  if (!tKeyCopy) {
+    return context.report({
+      node,
+      messageId: 'tKeyNotFoundInTranslations',
+      data: { tKey }
+    })
+  }
+
+  validatePlaceholdersAndValues(
+    node,
+    context,
+    getUniquePlaceholdersFromTKey(tKeyCopy),
+    getKeysFromValuesObjectExpression(objectValuesWrapper)
+  )
+
+  return null
+}
+
+const validateTranslationKey = (node, context) => {
+  const translationKeyAttribute = getJSXAttribute(node, 't')
+
+  if (translationKeyAttribute) {
+    if (translationKeyAttribute.value.type !== 'Literal') {
+      return context.report({
+        node: translationKeyAttribute,
+        messageId: 'tKeyMustBeStringLiteral'
+      })
+    }
+
+    const tKey = translationKeyAttribute.value.value
+    const tKeyCopy = getTranslations(context)[tKey]
+
+    if (!tKeyCopy) {
+      return context.report({
+        node: translationKeyAttribute,
+        messageId: 'tKeyNotFoundInTranslations',
+        data: { tKey }
+      })
+    }
+
+    validatePlaceholdersAndValues(
+      node,
+      context,
+      getUniquePlaceholdersFromTKey(tKeyCopy),
+      getKeysFromValuesObjectExpression(getValuesObjectFromJSX(node))
+    )
+  }
+
+  return null
+}
+
+const validateValuesObject = (context, valuesObject) => {
+  if (valuesObject !== null) {
+    if (valuesObject.type !== 'ObjectExpression') {
+      return context.report({
+        node: valuesObject,
+        messageId: 'valuesMustBeObject'
+      })
+    }
+
+
+    valuesObject.properties.forEach(property => {
+      if (property.computed) {
+        context.report({
+          node: property,
+          messageId: 'valuesKeyMustBeStringLiteral'
+        })
+      }
+    })
+  }
+
+  return null
+}
+
+const isTranslationComponent = node =>
+  ['Translate', 'TranslateRef', 'TranslateNode', 'Text', 'TextNode'].includes(
+    node.name.name
+  )
+
+const getValuesObjectFromJSX = node => {
+  const attribute = getJSXAttribute(node, 'values')
+
+  if (!attribute) {
+    return null
+  }
+
+  return attribute.value.expression
+}
+
+const getValuesObjectFromFunctionCall = node => {
+  if (node.arguments.length < 2) {
+    return null
+  }
+
+  return node.arguments[1]
+}
+
+export const correctTranslationUsage = {
+  schema: [
```

Comment:
`schema` should be inside `meta`, no? https://eslint.org/docs/developer-guide/working-with-rules#options-schemas